crates/task-impls/src/quorum_proposal_recv/handlers.rs

```diff
@@ -90,55 +98,41 @@ async fn validate_proposal_liveness<TYPES: NodeType, I: NodeImplementation<TYPES
     .await;
 
     if liveness_check {
-        let new_view = proposal.data.view_number + 1;
```

Comment:
Sure, I guess my question though is: where do we perform this check? It seems like it should still be occurring, right? If not, then when the parent state is not found, we never attempt to propose if the liveness check passes.

Reply:
Spoke offline, we propose anyway because the validity path is not needed to be able to submit a proposal anyway, so we don't need the explicit livenesscheck->proposal flow. Thanks for clarifying!